beta/src/pages/learn/index.md

```diff
@@ -389,27 +388,33 @@ Hooks are more restrictive than regular functions. You can only call Hooks *at t
 
 ## Sharing data between components {/*sharing-data-between-components*/}
 
-In the previous example, each button had its own independent counter:
+In the previous example, each `MyButton` had its own independent `count`, and when each button was clicked, only the `count` for the button clicked changed:
 
-```js {2-4}
-- MyApp
-  - MyButton (count: 3)
-  - MyButton (count: 1)
-  - MyButton (count: 2)
-```
+<DiagramGroup>
+    <Diagram name="sharing_data_child.svg" alt="TODO: Sharing data" height={734} width={814}>
+        Before clicking, each MyButton has a count value set to zero.
+    </Diagram>
+    <Diagram name="sharing_data_child_clicked.svg" alt="TODO: Sharing data" height={734} width={814}>
+        After clicking, only one MyButton count value has updated.
+    </Diagram>
+</DiagramGroup>
 
-However, you'll often need components to *share data and always update together*.
+However, often you'll need components to *share data and always update together*.
 
-To make all buttons display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them. In this example, it is `MyApp`:
+To make both `MyButton` components display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them.
 
-```js {1}
-- MyApp (count: 3)
-  - MyButton
-  - MyButton
-  - MyButton
-```
+In this example, it is `MyApp`:
+
+<DiagramGroup>
+    <Diagram name="sharing_data_parent.svg" alt="TODO: Sharing data" height={770} width={820}>
+        Before clicking, count is stored in MyApp and passed down to both children as props.
```

Comment:
I think you should be able to write this without indentation. We generally prefer that in MDX. But you need newlines. So,

```js
<DiagramGroup>

<Diagram>

Before ...

</Diagram>

...

```

Reply:
Should we backtick technical names like `count` in the captions?

Reply:
Do you have a tip for how to make that work? Since it's a prop string and not markdown the back ticks just show as back ticks.

Reply:
Right, it doesn't work because there are no newlines before and after. (MDX 1 is a bit confusing in that way.) If you format it like I wrote above, you should be able to use backticks inside.